app/controllers/posts_controller.rb

```diff
@@ -10,4 +10,31 @@ def show
     @post = Post.find(@user_id)
     @comments = @post.comments
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
+    @post.author = current_user
+
+    if @post.save
```

Comment:
- [ ] As requested in the [previous](https://github.com/addisalemseifu/my_blog_app/pull/7#discussion_r1338429605) review, I attempted to create a post from the page; however, I noticed that this issue has not been addressed, and upon further inspection, it appears that comments and likes counters are still missing from the post item. Kindly prioritize the resolution of this matter. :wink: